src/mlpack/methods/ann/init_rules/glorot_init.hpp

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file glorot_init.hpp
+ * @author Prabhat Sharma
+ *
+ * Definition and implementation of the Glorot initialization method. This
+ * initialization rule initialize the weights to maintain activation variances
+ * and back-propagated gradients variance as one moves up or down the network.
+ *
+ * For more information, see the following paper.
+ *
+ * @code
+ * @inproceedings{pmlr-v9-glorot10a,
+ * title = 	 {Understanding the difficulty of training deep feedforward neural networks},
+ * author = 	 {Xavier Glorot and Yoshua Bengio},
+ * booktitle = 	 {Proceedings of the Thirteenth International Conference on Artificial Intelligence and Statistics},
```

Comment:
Minor style comment:
I think you can avoid whitespaces around `=`
And please make sure lines are under 80 characters.